metadata.json

```diff
@@ -20,6 +20,10 @@
     {
       "name": "simp/simplib",
       "version_requirement": ">= 3.6.0 < 5.0.0"
+    },
+    {
+      "name": "herculesteam/augeasproviders_grub",
+      "version_requirement": ">= 2.4.0 < 4.0.0"
```

Comment:
The `kernel_parameter` type was added before 2.4.0, but 2.4.0 is still probably the minimum due to your change in https://github.com/hercules-team/augeasproviders_grub/commit/86fca6f5b9dbdf16a3f714270b36137fd8b4bb58#diff-e9cebb996393d3df6f9bf4605677765eR11